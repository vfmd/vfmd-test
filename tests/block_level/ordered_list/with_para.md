1. List item
   List item
   Part of the list

1. List item
   List item
Part of the list

1. List item

Not part of the list
