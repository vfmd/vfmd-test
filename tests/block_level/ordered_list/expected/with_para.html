<ol>
<li><p>List item
List item
Part of the list</p></li>
<li><p>List item
List item
Part of the list</p></li>
<li><p>List item</p></li>
</ol>

<p>Not part of the list</p>
